styx-common/src/main/java/com/spotify/styx/storage/DatastoreStorage.java

```diff
@@ -849,8 +849,13 @@ static Backfill entityToBackfill(Entity entity) {
         .nextTrigger(timestampToInstant(entity.getTimestamp(PROPERTY_NEXT_TRIGGER)))
         .schedule(Schedule.parse(entity.getString(PROPERTY_SCHEDULE)))
         .allTriggered(entity.getBoolean(PROPERTY_ALL_TRIGGERED))
-        .halted(entity.getBoolean(PROPERTY_HALTED))
-        .reverse(Optional.ofNullable(entity.getBoolean(PROPERTY_REVERSE)).orElse(false));
+        .halted(entity.getBoolean(PROPERTY_HALTED));
+
+    if (entity.contains(PROPERTY_REVERSE)) {
+      builder.reverse(entity.getBoolean(PROPERTY_REVERSE));
```

Comment:
`.reverse(entity.contains(PROPERTY_REVERSE) && entity.getBoolean(PROPERTY_REVERSE))` would this be more fluent or actually harder to read?

Reply:
I wanted to be consistent with how description was handled. But indeed, personally I find the if-else easier and quicker to understand and using the boolean logic

Reply:
Use `read()`?

```java
.reverse(read(entity, PROPERTY_REVERSE, Boolean.FALSE))
```

Reply:
`read()` is an object method, can't access it from the static `entityToBackfill()`. I'll end up doing 
```
DatastoreStorage.<Boolean>readOpt(entity, PROPERTY_REVERSE).orElse(Boolean.FALSE)
```
instead

Reply:
Feel free to change `entityToBackfill()` to not be static or make `read()` static as well.